tests/integration/test_integration.py

```diff
@@ -481,7 +481,8 @@ def test_call_basic_config(tmpdir, call_basic_config, debug_mode):
 print(len(root.handlers))
 """.lstrip()
     )
-    env = {}
+    env = os.environ.copy()
```

Comment:
was this the fix for the `ddtrace-run cannot be found` error?

Reply:
Almost certainly because I think the `PYTHONPATH` needs to be passed

Reply:
I'm curious why this was unnecessary for the other tests that run `ddtrace-run` subprocess.Popen commands...

Reply:
yeah, exactly, because `env` to `Popen` isn't additive, it replaces the env.

Reply:
> I'm curious why this was unnecessary for the other tests that run ddtrace-run subprocess.Popen commands...

I assume because when not supplying `env` it just inherits.

Reply:
https://docs.python.org/3/library/subprocess.html#subprocess.Popen

> If env is not None, it must be a mapping that defines the environment variables for the new process; these are used instead of the default behavior of inheriting the current process’ environment.